<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="nameRecipientLabel">
        <property name="text">
         <string>Recipient's name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameRecipientLineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="messageLabel">
        <property name="text">
         <string>Message:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="messageLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="sendPushButton">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QTextBrowser" name="commonChatBrowser"/>
      <widget class="QTextBrowser" name="privateChatBrowser"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>693</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain_menu">
    <property name="title">
     <string>Main menu</string>
    </property>
    <addaction name="actionOpenAnotherClient"/>
    <addaction name="separator"/>
    <addaction name="actionCloseClient"/>
   </widget>
   <addaction name="menuMain_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenAnotherClient">
   <property name="text">
    <string>Open another client</string>
   </property>
  </action>
  <action name="actionCloseClient">
   <property name="text">
    <string>Close this client</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
